src/pages/using-spark/guides/writing-meaningful-web-content-with-semantic-html.mdx

```diff
@@ -0,0 +1,217 @@
+    </li>
+    <li>
+      <a href="#common-html-elements-for-ux-writers">
+        Common HTML Elements For UX writers
+      </a>
+    </li>
+  </ul>
+  <li>
+    <a href="#semantic-elements">Semantic Elements</a>
+  </li>
+  <ul class="sprk-b-List sprk-b-List--indented sprk-u-Measure sprk-u-mvs">
+    <li>
+      <a href="#the-strong-element">The Strong Element</a>
+    </li>
+    <li>
+      <a href="#strong-vs-bold-vs-css-styling">Strong vs Bold vs CSS Styling</a>
+    </li>
+    <li>
+      <a href="#the-emphasis-element">The Emphasis Element</a>
+    </li>
+    <li>
+      <a href="#emphasis-vs-italic">Emphasis vs Italic</a>
+    </li>
+  </ul>
+  <li>
+    <a href="#accessibility">Accessibility</a>
+  </li>
+</ul>
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvn" />
+
+## HTML Elements
+
+HTML consists of series of elements, which enclose, or wrap pieces of content they appear or act a certain way.
+
+Elements describe or represent the meaning, structure, or context to web browsers, developers, and users of assistive technologies.
+
+> For more information, visit <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element"> Mozilla's Developer Documentation - HTML elements</a>.
+
+As a UX Writer, it's important to:
+
+- **Understand** what is the purpose of the enclosed text.
+- **Know** what element to use depending on the purpose of the text.
+- **Ensure** we are communicating the intended meaning of the enclosed text to the users.
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvn" />
+
+### Creating Elements Using Tags
+
+Use **HTML tags** to create an element. These tags represent the start and end of a piece of content.
+
+An element consists of three main parts:
+
+- An **opening tag** that represents the start of the paragraph element.
+- **Content** within the tags.
+- A **closing tag** that represents where the element ends.
+
+<h3 class="sprk-b-TypeDisplayFour sprk-u-mbm sprk-u-Measure">Example</h3>
+
+`<p>` Download the Rocket Homes mobile application on your mobile device to start looking for your dream home.`</p>`
+
+- The `<p>` represents the start of the paragraph element.
+- The content is the enclosed text (<i>"Download the Rocket.... " </i>) inside the tags.
+- The `</p>` represents where the element ends.
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvn" />
+
+### Common HTML Elements For UX Writers
+
+Use the strong and emphasis elements, to ensure users can perceive the semantic meaning indicated by the visual treatment, even if they’re using assistive technology like a screen reader.
+
+- The **strong element** `<strong></strong>`, defines text with strong importance.
+- The **emphasis element** `<em></em>`, defines text with stressed emphasis.
+
+> As a general rule, **don't use** the **bold** or the **italic** elements if there is a more descriptive and relevant tag available. Instead, use for formatting and to display special types of text.
```

Comment:
a little bold heavy here, and the sentence ends awkwardly.

`> As a general rule, avoid using **bold** or **italic** elements because they do not add semantic meaning. Do not rely on HTML to create a visual change, and collaborate with developers to correctly emphasize content.`